package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "protractor-flake",
   "description": "A wrapper for protractor to automatically re-run failed specs for a specific number of attempts",
+  "version": "1.0.4",
```

Comment:
This is actually part of semantic-release, so please leave it out. It gets added when the package is published.